src/UserRepository.js

```diff
@@ -1,7 +1,32 @@
-class UserRepository {
-  constructor() {
+const data = require('../data/users.js');
+const userData = data.testData;
```

Comment:
I'm not sure we need to require this here, but on the main script instead, since thats where we will be instantiating our user repos

Reply:
adjusted